planner/optimizer/optimizer-core/src/main/java/eu/seaclouds/platform/planner/optimizer/util/YAMLmodulesOptimizerParser.java

```
@@ -350,4 +350,17 @@ private static QualityInformation parseQoS(Map<String, Object> qosInformation) {
       return availabilityMapValue;
    }
 
+   public static boolean getScalabilityCapabilitiesOfModule(Map<String, Object> modules, String elementName) {
+      try{
+       Map<String,Object> moduleInfo = (Map<String,Object>) modules.get(elementName);  
+       Map<String,Object> moduleProperties= (Map<String,Object>) moduleInfo.get(TOSCAkeywords.MODULE_PROPERTIES_TAG); 
+       boolean moduleCanScale = (boolean) moduleProperties.get(TOSCAkeywords.MODULE_AUTOSCALE_PROPERTY); 
+       return moduleCanScale;
+      }catch(Exception e){
```

Avoiding Empty catch. For example, you could use log